lib/UI/Express.js

```diff
@@ -251,6 +251,20 @@ class UIExpress extends Express {
 			}
 		})
 
+		this.get('^/get/custom-variable/:name', (req, res) => {
+			res.header('Access-Control-Allow-Origin', '*')
+			res.header('Access-Control-Allow-Methods', 'GET,OPTIONS')
+			res.header('Access-Control-Allow-Headers', 'Content-Type, Authorization, Content-Length, X-Requested-With')
+
+			this.logger.debug(`Got HTTP /get/custom-variable/ name ${req.params.name}`)
+			const result = this.registry.instance.variable.custom.getValue(req.params.name)
+			if (result) {
```

Comment:
```suggestion
			if (result === undefined) {
```
A variable could have a value of an empty string, which would be valid. Being `undefined` is what we want to catch here as it implies it doesnt exist